requirements.txt

```diff
@@ -3,3 +3,4 @@ tqdm>=4.59.0
 ruamel.yaml<=0.16.6
 tabulate
 numpy
+scikit-learn
```

Comment:
Is it necessary to add `scikit-learn`?  It can be imported as `from towhee.utils import sklearn_utils` like metricmixin.

Reply:
Cool